protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -553,10 +560,17 @@ func (k Keeper) internalCanUpdateSubaccounts(
 
 	// Block all withdrawals and transfers if either of the following is true within the last
 	// `WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS`:
-	// - There was a negative TNC subaccount seen.
+	// - There was a negative TNC subaccount seen for any of the collateral pools of subaccounts being updated
```

Comment:
for my understanding - we are blocking transfers both from and to the gated collateral pool, right? users also can't transfer to the pool

Reply:
Yes, that's correct. Users can deposit into the gated collateral pool though still.